Tasks/PublishTestResultsV1/publishtestresults.ts

```diff
@@ -19,8 +19,8 @@ tl.debug('publishRunAttachments: ' + publishRunAttachments);
 
 let matchingTestResultsFiles = ffl.findFiles(testResultsFiles, false, tl.getVariable('System.DefaultWorkingDirectory'));
 if(!matchingTestResultsFiles || matchingTestResultsFiles.length == 0) {
-  tl.warning('No test result files matching ' + testResultsFiles + ' were found.');  
-  tl.exit(0);
+  tl.warning('No test result files matching ' + testResultsFiles + ' were found.');
+  process.exit(0);
```

Comment:
@v-padiseshu How did you arrive using process.exit(0), What is it intended to do ?

Reply:
@upadhyayap In previous **vsts-tasks** library, There is a definition for **exit** tl.exit(0), But in **azure-pipelines-task-lib** it will not have a definition for exit. So as we verified in some other tasks we get to know about process.exit(). And after adding it i have deployed the task and tested. I didn't see any difference on tasks execution in pipeline

Reply:
Fixed now using **setResult**